packages/panels/resources/views/components/page/index.blade.php

```diff
@@ -132,6 +84,12 @@ class="fi-page-footer-widgets"
 
                 {{ \Filament\Support\Facades\FilamentView::renderHook('panels::page.footer-widgets.after', scopes: $this->getRenderHookScopes()) }}
             </div>
+
+            @if ($subNavigation && $subNavigationAlignment === Alignment::End)
+                <x-filament-panels::sidebar.sub-navigation
+                    :subNavigation="$subNavigation"
```

Comment:
Could you make sure to use kebab case for HTML attributes, @glennjacobs?

```suggestion
                    :sub-navigation="$subNavigation"
```

Reply:
Done